### PR TITLE
Allow publishing of bytes messages

### PR DESCRIPTION
**Pull request by PhearZero**
Useful for educational purposes such as serialization/encoding. (protobuf with minipb)

### CODE REVIEW

**Comment by brentru** [comment]
@PhearZero This is failing on PyLint, please refer to the guide here to fix it up: https://learn.adafruit.com/improve-your-code-with-pylint

I'll review once it's passing our build CI.

Also - you mention miniPB? Is there a CircuitPython library for this, or are you using MiniPB+MiniMQTT with CPython?

**Comment by PhearZero** [comment]
Sounds good I'll check out the lint error. Using Circuit/MicroPython(Preferably Circuit) to teach my wife about microcontrollers/messaging systems and it's been a great experience so far. 

 
> Also - this is for my curiosity - you mention miniPB? Is there a CircuitPython library for this, or are you using MiniPB+MiniMQTT with CPython?

I am using MiniPB + MiniMQTT in CircuitPython and passing a encoded bytes to my fork of MiniMQTT with this commit. For now, I commented out the MicroPython Logger in MiniPB and included the py source directly in my project.

**Review by brentru — APPROVED** [review]

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 1]
```diff
@@ -572,7 +572,7 @@ def publish(self, topic, msg, retain=False, qos=0):
             msg = str(msg).encode("ascii")
         elif isinstance(msg, str):
             msg = str(msg).encode("utf-8")
-        else:
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:575]: @PhearZero  Could you check `elif isinstance(msg, bytes)` and keep the `else:` after?

Comment (PhearZero) [reply adafruit_minimqtt/adafruit_minimqtt.py:575]: @brentru Good to go, learned a new thing today ```pass``` in Python. Hopefully that cleans it up.

**Review by brentru — APPROVED** [review]

File: adafruit_minimqtt/adafruit_minimqtt.py (hunk 1 above)
Comment (brentru) [reply adafruit_minimqtt/adafruit_minimqtt.py:575]: perfect.

**Pull request merged**